package.json

```
@@ -189,7 +189,7 @@
 		"eslint-plugin-jest-dom": "4.0.2",
 		"eslint-plugin-playwright": "0.8.0",
 		"eslint-plugin-ssr-friendly": "1.0.6",
-		"eslint-plugin-testing-library": "5.7.2",
+		"eslint-plugin-testing-library": "5.9.1",
```

Why this change in this PR?

---

Oh sorry that was unintentional
```suggestion
		"eslint-plugin-testing-library": "5.7.2",
```